Gemfile

```diff
@@ -1,5 +1,6 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
-gem 'rake', :require => false
-gem 'json', :require => false
-gem 'rubocop', :require => false
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```

Comment:
No idea, it was part of the skeleton generated by `bundle gem rzo`.  I'll remove it.